app/models/additional_funding_source.rb

```diff
@@ -1,13 +1,15 @@
 class AdditionalFundingSource < ApplicationRecord
   belongs_to :protocol
 
+  validates :funding_source, presence: true
   validates :fundig_source_other, presence: true, if: Proc.new { |a| a.funding_source == 'Internal Funded Pilot Project' }
```

Comment:
@6reg I see a typo here.  'fundig_source_other' instead of 'funding_source_other'.  Can you correct and resubmit this PR?